### ISSUE TITLE
Outline Control Modality Next Steps across Data, Training

### ISSUE THREAD

**Issue by harshsikka** [daniellawson9999/gato-control#59]
@jsjung00 is catching up to context from control modality work w/ help from @daniellawson9999. 

We want to rapidly understand what next datasets are needed, as we are finalizing the v0 benchmark in #58. We also want to start tracking research & implementation tasks across the control objective

**Comment by harshsikka** [comment]
@jsjung00 can we have a preliminary idea of what's next on this by Sunday? a brief slide or two on your thoughts after your sync w/ @daniellawson9999 would be great.

**Comment by jsjung00** [comment]
Timing wise I think Daniel and I would sync sometime after the end of the
work week, so hopefully this weekend.
Happy to present what we've talked about during the meeting if we meet
before then.

Best regards,
Justin

**Comment by harshsikka** [comment]
@jsjung00 makes sense! I just put it as a tentative deadline in case you are able to sync w/ Daniel ahead of time.
